Procfile

```
@@ -1,3 +1,3 @@
 web: bin/run
-worker: php bin/console messenger:consume async --time-limit=300 --memory-limit=256M --limit=50
+worker: bash -c "while true ; do php bin/console messenger:consume async --memory-limit=256M --limit=50 ; if [ $? -ne 0 ]; then break fi ; done"
```

Je suis perplexe... A quoi sert cette complexité ?

---

Ils m'ont suggéré une alternative plus simple

```suggestion
worker: bash -c "set -e; while true ; do php bin/console messenger:consume async --memory-limit=256M --limit=50 ; done"
```

En soi cette approche me va très bien, l'alternative serait de mettre des options dans l'UI mais là on gère le comportement voulu via le Procfile, et d'une façon qui semble assez robuste je trouve